src/lib/serviceRoutinesV2/postRegistration.cpp

```diff
@@ -0,0 +1,107 @@
+/*
+*
+* Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Ken Zangelin
+*/
+#include <string>
+#include <vector>
+
+#include "logMsg/logMsg.h"
+
+#include "common/defaultValues.h"
+#include "common/statistics.h"
+#include "common/clockFunctions.h"
+#include "common/string.h"
+#include "rest/ConnectionInfo.h"
+#include "rest/OrionError.h"
+#include "ngsi/ParseData.h"
+#include "apiTypesV2/Registration.h"
+#include "mongoBackend/mongoRegistrationCreate.h"
+#include "serviceRoutinesV2/postRegistration.h"
+#include "alarmMgr/alarmMgr.h"
+
+
+
+/* ****************************************************************************
+*
+* postRegistration - 
+*
+* POST /v2/registrations
+*
+* Payload In:  ngsiv2::Registration
+* <TMP>
```

Comment:
This is a new mark you have just invented? :)

Maybe un "FIXME PR remove old stuff" could be used (that "PR" would not refer to the present PR but to the merging PR of the feature/ngsiv2_registrations into master)

Reply:
Yeah, I have been using <TMP></TMP> for years ... :-)
This time I forgot to look for it before PR :-(
I remove it

Reply:
Fixed in 1a49367